File: API/models/student.js
const mongoose = require('mongoose');
let Schema = mongoose.Schema;

const studentSchema = new Schema({
    fullname:{
        type: String, required: true,
    },
    _id:{
        type: String, required: true,
    },
    password:{
        type: String, required: true,
    },
    email:{
        type: String, required: true, unique: true
    },
    level:{
        type: Array, required: false
    },
    followings:{
        type: Array, default: []
    },
    followers:{
        type: Array, default: []
    }
},
{
    collection: "students"
}); 

module.exports = mongoose.model("Student", studentSchema);

File: CodeWizUnity/Assets/API/Student.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class Student
{
    public string _id;
    public string username;
    public string fullname;
    public string password;
    public string email;
    [SerializeField]
    public Level[] level;
    public int __v;

    public override string ToString()
    {
        return _id;
    }
}




